INSTALL.md

```diff
@@ -342,6 +344,64 @@ ninja pycore
 ninja qgis_desktop
 ```
 
+### 3.7.1 Available compilation flags
+
+QGIS built is tunable according to your needs. Many flags are available to activate or deactivate some functionalities:
+
+* `WITH_3D`: Determines whether QGIS 3D library should be built
+* `WITH_ANALYSIS`: Determines whether QGIS analysis library should be built
+* `WITH_APIDOC`: Determines whether the QGIS API doxygen documentation should be built
+* `WITH_ASAN`: Use AddressSanitizer
+* `WITH_ASTYLE`: If you plan to contribute you should reindent with scripts/prepare_commit.sh (using 'our' astyle)
+* `WITH_AUTH`: Determines whether QGIS authentication methods should be built
+* `WITH_BINDINGS`: Determines whether python bindings should be built
+* `WITH_CLANG_TIDY`: Use Clang tidy
+* `WITH_CLAZY`: Determines whether Clazy checks should be enabled
+* `WITH_COPC`: Determines whether Cloud Optimized Point Cloud (COPC) support should be built
+* `WITH_CORE`: Determines whether QGIS core should be built.
+* `WITH_CRASH_HANDLER`: Determines whether the QGIS crash handler application should be built
+* `WITH_CUSTOM_WIDGETS`: Determines whether QGIS custom widgets for Qt Designer should be built
+* `WITH_DESKTOP`: Determines whether QGIS desktop should be built
+* `WITH_EPT`: Determines whether Entwine Point Cloud (EPT) support should be built
+* `WITH_GRASS${GRASS_CACHE_VERSION}`: Determines whether GRASS ${GRASS_SEARCH_VERSION} plugin should be built
+* `WITH_GUI`: Determines whether QGIS GUI library (and everything built on top of it) should be built
+* `WITH_HANA`: Determines whether SAP HANA Spatial support should be built
+* `WITH_INTERNAL_${ulib}`: Determines whether python ${lib} should be included
+* `WITH_INTERNAL_LAZPERF`: Determines whether LazPerf should be built from internal copy
+* `WITH_INTERNAL_MDAL`: Determines whether MDAL should be built from internal copy
+* `WITH_INTERNAL_POLY2TRI`: Determines whether POLY2TRI should be built from internal copy
+* `WITH_INTERNAL_QWTPOLAR`: Use internal build of QwtPolar
+* `WITH_OAUTH2_PLUGIN`: Build OAuth2 authentication method plugin
```

Comment:
```suggestion
* `WITH_OAUTH2_PLUGIN`: Determines whether OAuth2 authentication method plugin should be built
```

Homogenization

Reply:
I am agree but this text is extracted from CMakeLists.txt, should I apply your suggestion?

Reply:
Based on Nyall's review, I would say you are free to change it if you want ! 😀

Reply:
I will modified the INSTALL.md and CMakeLists.txt accordingly